src/doc/en/developer/packaging_sage_library.rst

```diff
@@ -472,6 +472,17 @@ requiring all of Sage to be present.
 mechanism mentioned above can also be used for this.
 
 
+Dependencies of the Sage documentation
+--------------------------------------
+
+The documentation will not be modularized.
+
+However, some parts of the Sage reference manual may depend on functionality
+provided by optional packages. These portions of the reference manual
+should be conditionalized using the Sphinx directive ``.. only ::``,
+as explained in :ref:`section-documentation-conditional`.
```

Comment:
``.. only::``

We have used (for instance, in `src/sage/plot/plot3d/plot3d.py`)
```
.. ONLY:: html                                        
```
So 
```
.. ONLY::
```
?

Reply:
Thanks, done

Reply:
Not done in `src/doc/en/developer/packaging_sage_library.rst`?

Reply:
Sorry, fixed now in de208b1

Reply:
No space between ``ONLY`` and ``::``

Reply:
Thanks. Fixed in 3f3469e